app/routes/git_tokens.py

```
@@ -1,204 +1,343 @@
+
+@router.get(
+    "/{label}",
+    response_model=Dict[str, Any],
+    status_code=status.HTTP_200_OK,
+    summary="Get git labels by label",
+    description="Retrieve git labels matching the specified label with masked token values",
+)
+async def get_git_label_by_label(
+    label: str,
+    current_user_id: str = Depends(get_current_user_id),
+    pagination: PaginationParams = Depends(),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)